getting_started.md

```diff
@@ -8,11 +8,15 @@ The goal of this document is get the site and backend working locally on your de
 
 ### Public Site
 
+Install `node >= 12.x.x` and `npm`: https://nodejs.org/en/download/.
```

Comment:
Can local-setup.sh detect this?

Reply:
Do you mean detect where it's installed and throw or install automatically if not the correct version?

Reply:
Just wondering what sort of error you'd get, and when, if node < 12 was installed.

Reply:
you'd get some kind of "npm undefined" message if not installed and then you'd get something like this for node < 12: https://stackoverflow.com/questions/45088031/how-to-ignore-incompatible-engine-node-error-on-installing-npm-dependencies-wi